src/pages/components/data-table/usage.mdx

```diff
@@ -277,148 +280,312 @@ import {
+
+<Row>
+<Column colLg={12}>
+
+![data table multi select](images/datatable-multi-select.png)
+
+</Column>
+</Row>
+
+### Inline actions
+
+Inline actions are functions that may be performed on a specific table row. Each row is accompanied by an [overflow menu](https://www.carbondesignsystem.com/components/overflow-menu/code) that contains actions related specifically to that row.
+
+<Row>
+<Column colLg={8}>
+
+![data table inline actions](images/datatable-inline-actions.png)
+
+</Column>
+</Row>
+
+When the overflow menu contains less than three options, keep the actions inline as icon buttons instead. This reduces a click and makes available actions visible at a glance.
+
+### Overflow menu
+
+By default, the overflow menu icons are persistent on each row. Having the overflow menus always visible signals to the user actions can be taken on the table rows. Alternatively, the overflow menu can be set to only show on hover and focus using the `overflowMenuOnHover` prop.
+
+<Row>
+<Column colLg={8}>
+
+![data table inline actions](images/datatable-persistent-hover.png)
+
+</Column>
+</Row>
+
+### Expandable
+
+The expandable data table is useful for presenting large amounts of data in a small space. Use the expanded section for supplementary information or data that needs additional query time. When the content in the expanded area begins to feel cramped, consider taking the user to a dedicated page or opening a modal to view the information and complete tasks.
+
+<Row>
+<Column colLg={12}>
+
+![data table in expanded and collapsed state](images/datatable-expand.png)
 
 <Caption>Expandable data table with one row expanded</Caption>
 
-By default, the expand-all chevron is not shown. Keeping detailed information in expanded section saves user loading time by postponing some data queries until needed. Expanding all rows at once will typically negate this advantages of the expandable table. If additional load time is expected to display expanded information, use skeleton states instead of spinners.
+</Column>
+</Row>
 
-<div className="image--fixed">
+By default, the expand-all chevron is not shown. Keeping detailed information in expanded section saves user loading time by postponing some data queries until needed. Expanding all rows at once will typically negate this advantages of the expandable table. If additional load time is expected to display expanded information, use skeleton states instead of spinners.
 
-![Expandabling all rows with skeleton states](images/data-table-usage-12.png)
+<Row>
+<Column colLg={12}>
 
-</div>
+![data table in expanded and collapsed state](images/datatable-skeleton.png)
 
 <Caption>Expanding all rows with skeleton states</Caption>
 
-## Feedback
+</Column>
+</Row>
+
+## Content
```

Comment:
In the new updates to the template the content section should go after the "formatting" section and before "behaviors"

Reply:
I'll update to the latest template format. This change pushes the behaviors section further down on the usage tab. Wouldn't the content section be better on the style tab?